pydoof/search_api/enums.py

```diff
@@ -0,0 +1,14 @@
+from enum import Enum, unique
+
+
+@unique
+class QueryNames(Enum):
+    MATCH_AND = "match_and"
+    MATCH_OR = "match_or"
+    FUZZY = "fuzzy"
+
+
+@unique
+class SearchFilterExecution(Enum):
+    AND = "and"
+    OR = "or"
```

Comment:
This isn't something used across many modules, so I don't see the point in moving this to a new module. What is your idea?

Reply:
It's used on `search_api.search` and `search_api.test_search`. I create this file to have all the `Enums` of `search_api` module in one place. If you need to import one you will be 100% sure that it will be on `enums.py.`

As an alternative I suggest to create a new file named something like `custom_type_hints.py` and set enums, dataclass, etc. on it.

In my personal opinion is easier to maintain smaller files than one with all code, types, etc. in the same file.

@mdogo and @sonic182 you have more experience here than me. I would adapt to the "Doofinder way".

Reply:
I don't care if it is at `enums.py` or `search.py`

It may vote for @mdogo option if these classes are only used in that module (the test_search can import for this module instead of `enums.py` and it will not be horrible)

Reply:
The thing is that the enums we use are used only by one context. For instance, management API stats module has enums that are only used in a stats' context, the same with the search enums, etc. Maybe one day we will have to use some enums that are used across multiple context, and then we will need a `pydoof.enums` or something like that, but for the moment I think it is ok this way.